stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -32,6 +32,9 @@ spec:
           {{- if and (contains "0.9" .Values.controller.image.tag) .Values.controller.publishService.enabled }}
             - --publish-service={{ template "controller.publishServicePath" . }}
           {{- end }}
+          {{- if (not (eq .Values.controller.defaultSSLCertificate "")) }}
+            - --default-ssl-certificate={{ .Values.controller.defaultSSLCertificate }}
```

Comment:
I don't see that `controller.defaultSSLCertificate` is documented in the README.md file with the rest of the properties. Can you add an entry there for it?

Reply:
@mattfarina I've updated the commit to have the corresponding documentation in the README.md